packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -87,6 +87,7 @@ function network.primary_address(offset)
 	if invalid then
 		ipv4_template = mc:maxhost()
 		ipv4_template:prefix(tonumber(ipv4_maskbits))
+		print("INVALID IPv4 DETECTED, CHECK YOUR SETTINGS. USING " ..tostring(ipv4_template))
```

Comment:
I would put in the log message the exact name of the option line network.ipaddr or similar